### PR TITLE
Fix ci publish

### PR DESCRIPTION
**Pull request by job13er**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #none# - documentation fixes and/or test additions
 - [ ] #patch# - backwards-compatible bug fix
 - [x] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
* **Changed** the commit message for bump commits to no longer include `[ci skip]` in the commit message, this means that version bump commits will now trigger additional travis builds. If this works, I'll update the `README.md` with instructions on how to support `#none#` properly in your travis config.

### CODE REVIEW

File: lib/bumper.js [hunk 1]
```diff
@@ -50,40 +56,48 @@ class Bumper {
       return Promise.resolve()
     }
 
-    return this._getMergedPrInfo()
+    return this.vcs.getLastCommitMsg()
+      .then((commitMessage) => {
+        if (commitMessage === 'Automated version bump') {
+          throw new Cancel('Skipping bump on version bump commit.')
+        }
+        return Promise.resolve()
```
Comment (sandersky) [inline lib/bumper.js]: You don't actually need to return anything here, right?

**Comment by sandersky** [comment]
:+1:

File: lib/bumper.js (hunk 1 above)
Comment (job13er) [inline lib/bumper.js]: I'm not positive. I know if you return a primitive value (not a promise) it ends up returning a promise that resolves as that value, but I'm not sure about `undefined`, it's probably not necessary. I was chasing down an issue where something didn't have a `.then()` so I was being extra cautious. 

You're probably right though, I don't think it's needed.

**Comment by sandersky** [comment]
:+1:

**Pull request merged**